apps/web/pages/event-types/[type].tsx

```diff
@@ -1334,6 +1351,120 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                           </div>
                         </div>
 
+                        <>
+                          <hr className="border-neutral-200" />
+                          <div className="block flex-col sm:flex">
+                            <Controller
+                              name="seatsPerTimeSlot"
+                              control={formMethods.control}
+                              render={() => (
+                                <CheckboxField
+                                  id="seats"
+                                  name="seats"
+                                  label={t("offer_seats")}
+                                  description={t("offer_seats_description")}
+                                  defaultChecked={eventType.seatsPerTimeSlot !== null}
+                                  onChange={(e) => {
+                                    if (e?.target.checked) {
+                                      setEnableSeats(true);
+                                      // Want to disable individuals from taking multiple seats
+                                      formMethods.setValue("seatsPerTimeSlot", 2);
+                                      formMethods.setValue("disableGuests", true);
```

Comment:
If checked the minimum number of seats should be 2 and disable guests

Reply:
Here too try to add context to `2`, `const defaultSeatsPerTimeSlot = 2;`